src/components/select/select-textbox/select-textbox.component.js

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import I18n from 'i18n-js';
+import Textbox from '../../../__experimental__/components/textbox';
+import OptionsHelper from '../../../utils/helpers/options-helper/options-helper';
+
+const SelectTextbox = ({
+  value,
+  disabled,
+  readOnly,
+  placeholder,
+  onClick,
+  onFocus,
+  onBlur,
+  selectedValue,
+  ...restProps
+}) => {
+  const defaultPlaceholder = I18n.t('select.placeholder', {
+    defaultValue: 'Please Select...'
```

Comment:
This should be a `defaultProp`
```suggestion
    defaultValue: placeholder
```

Reply:
it's the default value of the translation, if we change it then the user's placeholder wouldn't be replaced with default translation?

Reply:
You're right, your implementation allows for a user to override the translation with a placeholder. That's more beneficial.